tools/maintainers/README.md

```diff
@@ -3,9 +3,9 @@
 Changes to team membership should happen in this way:
 
 * Follow [community
-  governance](https://github.com/instructlab/community/blob/main/governance.md)
+  governance](../../GOVERNANCE.md)
```

Comment:
I don't think we generally use relative links. Always full links. The full links always point to the main branch of the main repo rather than some branch in some repo. 

Can you please restore the full links with any needed corrections?

Reply:
@bjhargrave sorry for the delayed response, totally missed the notification. 

Sure, i can revert it to full link, but do we maintain multiple version of the repo? IMHO, I think if we use the relative links, we can catch these kind of issue (broken link to the docs within the repo),  during the markdown linting itself, to ensure repo documents doesn't have any broken link. So i think relative links help us more here compared to full links. wdyt?

Reply:
A large issue with relative links is that they are relative. They point into whatever fork they are in and not the authoritative page in the main branch of the main repo.

Reply:
> I don't think we generally use relative links. Always full links.

Where do you take this from? I don't believe we have such a rule and the default ought to be to use relative links rather than absolute ones actually.

> A large issue with relative links is that they are relative. They point into whatever fork they are in and not the authoritative page in the main branch of the main repo.

:-) That's not a bug though, it's a feature! Indeed I would argue that in general a relative link ensures integrity of the link with regard to the state of the repo or clone. But I could agree that in this particular instance it might be better to use an absolute URL. I just don't think that's true for all links.

Reply:
> Where do you take this from? I don't believe we have such a rule and the default ought to be to use relative links rather than absolute ones actually.

I did not say there was a rule. I was referring to actual links in the docs which are almost always absolute. Perhaps there should be a rule and then it should be consistently applied.

Reply:
@bjhargrave Please correct me if I am wrong, but currently we just maintain the main branch, and i am not sure if we will get into a situation where we will have to maintain multiple branches of the community repo. Even if we get to the situation where we need to maintain multiple branches, every fork will have a copy of master anyways, and if the document changes in the branch (in this case governance.md), ideally the rest of the documents should point to the local copy of the governance.md and not the master one? If the governance.md in master is evolving, and we want all branches to point to that, I think this eventually will result in mess (some doc pointing to local changes, some pointing to master repo etc). Using the relative links is in general considered good practice and can be helpful to avoid publishing broken links and keeping the sanity of docs cross referencing.

Just my 2 cents, happy to fix the pr the way maintainers want.